syntax/swift.vim

```diff
@@ -83,25 +83,31 @@ syntax keyword swiftKeywords
       \ if
       \ import
       \ in
+      \ indirect
       \ infix
       \ init
       \ inout
       \ internal
+      \ internal(set)
       \ is
       \ lazy
       \ let
       \ mutating
       \ nil
+      \ nonmutating
       \ operator
       \ optional
       \ override
       \ postfix
       \ prefix
       \ private
+      \ private(set)
```

Comment:
Have you tried these? I really thought I tried this before and the parens through it off.